src/main/java/sdmExam/Board.java

```diff
@@ -0,0 +1,34 @@
+package sdmExam;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Board {
+    private List<Cell> _plays = new ArrayList<>();
+
+    public Board(){
```

Comment:
- Rename the for loop local variables i,j to row,column;
- Make the inner for loop body to a single line (e.g. add a static method Cell.empty to which you pass a position, make use also of the Position.in static method to generate a Position instance)